src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptionConverters.java

```diff
@@ -234,6 +234,42 @@ public String getTypeDescription() {
     }
   }
 
+  /**
+   * Flag converter for canonicalizing a label (possibly with a "/..." suffix) and/or define by
```

Comment:
Why does this mention `--define` (also the comment on line 245). It doesn't look like this converter is used except for with `--experimental_propagate_custom_flag`.

Reply:
The option accepts both Starlark flags and `--define`s.